lib/Qi.pm

```diff
@@ -10,7 +14,9 @@ sub startup {
   my $config = $self->plugin('Config');
 
   #Model
-   $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(pg => sub { state $pg = Mojo::Pg->new(shift->config('pg')) });
+  $self->helper(sql => sub{ state $sql = SQL::Abstract->new});
```

Comment:
I wouldn't recommend putting this here; instead put it in your base class.  Only your models need access to this object, not the entire app.